ReadASCIIApp/src/ReadASCII.cpp

```diff
@@ -93,6 +74,15 @@ ReadASCII::ReadASCII(const char *portName, const char *searchDir, const int step
     setIntegerParam(P_RampOn, 0);
     setIntegerParam(P_LookUpOn, 0);
 
+    char localDir[DIR_LENGTH], dir[DIR_LENGTH];
+    getStringParam(P_DirBase, DIR_LENGTH, dir);
+    getStringParam(P_Dir, DIR_LENGTH, localDir);
+    strcat(dir, "/");
+    strcat(dir, localDir);
+    status = createParams(dir);
+
+    createParam(P_CurTempString, asynParamFloat64, &P_CurTemp);
+
     // Do no initialise P_SPOut here because it will be sent to the eurotherm
 
     lastModified = 0;
```

Comment:
The line below this says
```
fileBad = true; //Set so that program doesn't attempt to read file before base dir is set
```
however `createParams` reads the file. Does the comment just need removing?